tljh/configurer.py

```diff
@@ -17,10 +17,8 @@
 # User provided config is merged into this
 default = {
     'auth': {
-        'type': 'firstuseauthenticator.FirstUseAuthenticator',
-        'FirstUseAuthenticator': {
-            'create_users': False
-        }
+        'type': 'nativeauthenticator.NativeAuthenticator',
```

Comment:
I don't think we should switch the default yet! This PR can be just adding it as an option, and adding docs. We can then work on migration pathway, and then make it default.